aim/tools/cli/debug/hashtree.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2016 Cisco Systems
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import click
+import json
+
+from aim.common.hashtree import exceptions as h_exc
+from aim import context
+from aim.db import api
+from aim.db import tree_model
+from aim.tools.cli.groups import aimcli
+
+
+@aimcli.aim.group(name='hashtree')
+@click.pass_context
+def hashtree(ctx):
```

Comment:
Did realize that debug CLI commands also had UTs. Will add it in a follow-up change if that is ok with you.